docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,272 @@
+1. Send additional ICP utility tokens to the (existing) neuron's address.
+2. Refresh the neuron on the NNS governance canister, which tells the governance canister that the transfer in (1) happened, upon which the NNS governance will update the neuron's stake. 
+
+:::info
+Note that refreshing of a neuron will also update the neuron's age to account for the fact that the newly added tokens have no age.
+
+:::
+
+These steps happen analogously to the claiming of neurons above.
+Again, care is advised to makes sure that the right account is computed and used.
+When `ClaimOrRefreshNeuronFromAccount` is used, the NNS governance canister will automatically either claim the neuron if it doesn't exist yet or top it up if it exists.
+
+_Required permissions:_ Anyone can make this call and top up a neuron, even if they do not have any permissions on the neuron. No matter who makes the call to the governance canister, its controller is set by the arguments as mentioned above.
+
+### Modifying a neuron's state
+Recall from the section _Neuron attributes_, that a neuron has a dissolve delay and can be non-dissolving, dissolving, or dissolved.
+To switch between the diffent dissolve states or to increase the dissolve delay, use the `ManageNeuron` command `Configure`. 
+```
+type Configure = record { operation : opt Operation };
+type Operation = variant {
+  ...
+  StopDissolving : record {};
+  StartDissolving : record {};
+  IncreaseDissolveDelay : IncreaseDissolveDelay;
+  ...
+  SetDissolveTimestamp : SetDissolveTimestamp;
+};
+type IncreaseDissolveDelay = record {
+  additional_dissolve_delay_seconds : nat32;
+};
+type SetDissolveTimestamp = record { dissolve_timestamp_seconds : nat64 };
+```
+
+To move a neuron from non-dissovling into dissovling state, and start the timer, use `StartDissolving`. No additional argument are required.
+
+To move a neuron from dissolving into non-dissolving state, and stop the timer at the dissolve state that the neuron has, use `StopDissolving`.
+
+No matter in which dissolve state a neuron is, its dissolve delay can be increased up to a maximum of eight years.
+To do so, there are two options.
+You can use `IncreaseDissolveDelay` and specify by the argument `additional_dissolve_delay_seconds` by how many seconds the current remaining dissolve delay should be increased. For example, if a neuron has 6 months dissolve delay, choosing the argument 2 months (in seconds) for `IncreaseDissolveDelay`, would result in a neuron with 8 months dissolve delay. 
+Alternatively, you can specify how much dissolve delay a neuron should have after the operation by using `SetDissolveTimestamp` and setting a target time stamp `dissolve_timestamp_seconds` in seconds from the Unix epoch.
+To achieve the same as above, compute the timestamp of now plus 8 months and provide this as the input `dissolve_timestamp_seconds`. 
+In both cases, the dissolve delay can only be increased but never decreased.
+
```

Comment:
This is a bit funky and counterintuitive. In particular we're using this wrongly in Rosetta (afaiu) @MarioDfinity

Reply:
Is this just a comment for Mario to check in Rosetta but nothing to change in the text here?